client/me/account-close/main.jsx

```diff
@@ -108,7 +127,16 @@ class AccountSettingsClose extends Component {
 									<ActionPanelFigureListItem>
 										{ translate( 'Personal details' ) }
 									</ActionPanelFigureListItem>
-									<ActionPanelFigureListItem>{ translate( 'Sites' ) }</ActionPanelFigureListItem>
+									<ActionPanelFigureListItem className="account-close__sites-item">
+										{ translate( 'Sites' ) }
+										<Gridicon size={ 18 } onClick={ this.handleSiteDropdown } icon="chevron-down" />
+									</ActionPanelFigureListItem>
+									{ this.state.showSiteDropdown && (
+										<SiteSelector
+											filter={ this.siteListFilter }
+											noResultsMessage={ translate( 'You have no sites that will be deleted' ) }
```

Comment:
If we know we won't be deleting any sites, I wonder if we should just omit the 'Sites' item from the list altogether (along with related ones like posts, pages and media).

Reply:
I tried to get this working by somehow checking how many sites are filtered, or by trying to check if the `site-selector__no-results` class was on the page, but I couldn't get that working. Any ideas? :)